pennylane/qchem/convert_openfermion.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the functions for converting an openfermion fermionic operator to Pennylane
+FermiWord or FermiSentence operators
+"""
+# pylint: disable= import-outside-toplevel,no-member,unused-import
+from pennylane.fermi import FermiSentence, FermiWord
+
+
+def from_openfermion(of_op, tol=1e-16):
```

Comment:
I suggest `openfermion_op` to be consistent with `from_qiskit_op` function. The examples are fine, they can still use `of_op`.